test/Microsoft.AspNetCore.Mvc.RazorPages.Test/PageModelTest.cs

```diff
@@ -1903,8 +1949,8 @@ public void ViewComponent_WithName()
             {
                 PageContext = new PageContext
                 {
-                    ViewData = viewData,
```

Comment:
Undo

Reply:
You need me to leave the extra comma in the ViewComponent test?!!

Reply:
Yes please 👍

Reply:
Sure

Reply:
Done!! but i'm not sure what's the need for the extra comma!!